lib/devicetrust/native/device_windows.go

```diff
@@ -78,106 +79,90 @@ func handleTPMActivateCredential(encryptedCredential, encryptedCredentialSecret
 	return windowsDevice.handleTPMActivateCredential(encryptedCredential, encryptedCredentialSecret)
 }
 
-// getDeviceSerial returns the serial number of the device using PowerShell to
-// grab the correct WMI objects. Getting it without calling into PS is possible,
-// but requires interfacing with the ancient Win32 COM APIs.
 func getDeviceSerial() (string, error) {
-	cmd := exec.Command(
-		"powershell",
-		"-NoProfile",
-		"Get-WmiObject Win32_BIOS | Select -ExpandProperty SerialNumber",
-	)
 	// ThinkPad P P14s:
 	// PS > Get-WmiObject Win32_BIOS | Select -ExpandProperty SerialNumber
 	// PF47WND6
-	out, err := cmd.Output()
-	if err != nil {
+
+	type Win32_BIOS struct {
+		SerialNumber string
+	}
+
+	var bios []Win32_BIOS
+	query := wmi.CreateQuery(&bios, "")
+	if err := wmi.Query(query, &bios); err != nil {
 		return "", trace.Wrap(err)
 	}
-	return string(bytes.TrimSpace(out)), nil
+
+	if len(bios) == 0 {
+		return "", trace.NotFound("device serial not found")
```

Comment:
nit/minor: I would probably go with BadParameter just for fear that NotFound could be interpreted in other ways (like the device not existing server-side). Today this is inconsequential, as we swallow those errors anyway.

Reply:
Will do! Thanks for testing!